plugins/TapTempo/TapTempoUi.h

```diff
@@ -0,0 +1,195 @@
+/********************************************************************************
+** Form generated from reading UI file 'taptempo.ui'
```

Comment:
I guess you initially desinged the UI using Qt designer. You should probably cleanup UI class stuff.

Reply:
I mean, why this class should even exist apart from `TapTempoView`?

Reply:
When reading up on Qt's UI designer file documentation, this was one of the approaches to get it in code, which was using the ``setupUi`` function.

It's nice that I can separate the stylistic part of the UI into a separate UI file, and focus on only what I want to focus on in the plugin view file.

If I moved the UI into ``TapTempoView``, the whole file would be cluttered, making things harder to review and code.

Reply:
I think I agree with PhysSong on this one. `TapTempoView` and `TapTempoUi` should probably be combined into a single `TapTempoView` class in a `TapTempoView.h` file.

If the concern is over the clutter, it wouldn't be very cluttered if you created a corresponding source file `TapTempoView.cpp` to implement the methods currently in `TapTempoUi` rather than doing it all in the header.

Reply:
> I think I agree with PhysSong on this one. `TapTempoView` and `TapTempoUi` should probably be combined into a single `TapTempoView` class in a `TapTempoView.h` file.
> 
> If the concern is over the clutter, it wouldn't be very cluttered if you created a corresponding source file `TapTempoView.cpp` to implement the methods currently in `TapTempoUi` rather than doing it all in the header.

Just went ahead and merged the two files. Let me know if there is any further action needed or it's alright :+1:.

Reply:
I think it looks a lot better now! Other than the couple small issues I just pointed out in a review, everything looks fine to me coding-wise. I haven't looked into the logic or tested the plugin yet, but I may be able to do so tomorrow